test/connectors/ODSConnectorTest.scala

```diff
@@ -30,7 +32,9 @@ class ODSConnectorTest extends BaseSpec with WireMockHelper {
     new GuiceApplicationBuilder()
       .configure(
         "microservice.services.tax-summaries-hod.port" -> server.port(),
-        "microservice.services.tax-summaries-hod.host" -> "127.0.0.1"
+        "microservice.services.tax-summaries-hod.host" -> "127.0.0.1",
+        "play.ws.timeout.request"                      -> "1000ms",
+        "play.ws.timeout.connection"                   -> "500ms"
```

Comment:
Disable auditing to reduce noise

Reply:
good idea

Reply:
I used an application.conf for the tests it was quicker than changing all the test files